torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1532,7 +1541,11 @@ RegisterOperators reg2({
       Operator(                                                                \
           "prim::DictIndex(Dict(" key_type ", t) self, " key_type              \
```

Comment:
Now that dictionaries are mutable, all ops that extract values from the list (keys, values, index) will need to return wildcards.